OKTA-462650 Set servers parameter to optional ASA /preauthorizations (#3138)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

| `expires_at`   | string | The Preauthorization ceases to work after the `expires_at` date |
| `id`   | string | The UUID of the Preauthorization |
| `projects`   | array | The Projects that the Preauthorization is valid for |

> | `servers`   | array | Not supported; this field will be removed in a future release of ASA. It is only documented for backward compatibility with legacy clients. |

Reviewer: ```suggestion
| `servers`   | array | Not supported. This field will be removed in a future release of ASA. It's only documented for backward compatibility with legacy clients. |
```

| `expires_at`   | string | The Preauthorization ceases to work after the `expires_at` date |
| `id`   | string | The UUID of the Preauthorization |
| `projects`   | array | The Projects that the Preauthorization is valid for |

> | `servers`   | array | Not supported; this field will be removed in a future release of ASA. It is only documented for backward compatibility with legacy clients. |

Reviewer: ```suggestion
| `servers`   | array | Not supported. This field will be removed in a future release of ASA. It's only documented for backward compatibility with legacy clients. |
```
